src/html/runnabear.html

```diff
@@ -2,46 +2,110 @@
 <section id="bears" class="section">
 
   <h1 class="h1">Runnabear</h1>
-
   <h2 class="h2">Full Image</h2>
-
   <div class="figure-wrapper">
```

Comment:
I think this section would look nicer if the figures in each row were the same height (like in the screenshot below). You can do this using flexbox. Check out this [CodePen](http://codepen.io/lottejackson/pen/PwvjPj) for reference and let me know if you have any questions.

![image](https://cloud.githubusercontent.com/assets/7440805/24525231/64a99cf4-154e-11e7-86d4-dc2f08c590c7.png)